packages/foreman/rubygem-nokogiri/rubygem-nokogiri.spec

```diff
@@ -71,8 +71,9 @@ cp -a .%{gem_dir}/* \
         %{buildroot}%{gem_dir}/
 
 mkdir -p %{buildroot}%{gem_extdir_mri}
-cp -a .%{gem_extdir_mri}/gem.build_complete %{buildroot}%{gem_extdir_mri}/
-cp -a .%{gem_extdir_mri}/%{gem_name}/*.so %{buildroot}%{gem_extdir_mri}/
+cp -a .%{gem_extdir_mri}/%{gem_name}/gem.build_complete %{buildroot}%{gem_extdir_mri}/%{gem_name}
```

Comment:
I don't know if it needs to live in the subdirectory or not. Did you test this out?

Reply:
I did not but looking back at what was in the spec it looks like that's the wrong place for it to live. So I'm gonna change it.